blockprotocol/blockprotocol
# Upgrade `next-mdx-remote` and other markdown-related dependencies

Unblocks:

- https://github.com/blockprotocol/blockprotocol/pull/320


Was blocked by:

- `next-mdx-remote@v2` [is ESM-only](https://github.com/hashicorp/next-mdx-remote/releases/tag/v4.0.0), so we need to finish https://github.com/blockprotocol/blockprotocol/pull/322 first

Related links:

- [Asana task](https://app.asana.com/0/0/1202381398921853) _(internal)_
- [MDX2 migration guide](https://mdxjs.com/migrating/v2/)
- [next-mdx-remote@4.0.0 release notes](https://github.com/hashicorp/next-mdx-remote/releases/tag/v4.0.0)

How to test:

1. Open preview release `/docs` and `/specs`
1. Check visual consistency of inline code insertions, code blocks, info cards and also special anchor headers that start with ` ```block-function`.
